frontend/.env

```diff
@@ -0,0 +1 @@
+PORT=8082
```

Comment:
A lot of files are missing newlines at EOF. Can be easily fixed via IDE settings: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

Reply:
done!